apps/sms/views/conversation/js/conversation.js

```diff
@@ -1703,6 +1717,22 @@ var ConversationView = {
       pElement.setAttribute('data-l10n-id', 'no-attachment-text');
     }
 
+    var lateArrivalInfos = this.computeLateArrivalInfos(message, messageStatus);
+    if (lateArrivalInfos) {
+      var lateArrivalNoticeDOM = this.tmpl.lateArrivalNotice.prepare({})
```

Comment:
so, I checked the existing lib, and I think it actually already supports having no options. See https://github.com/mozilla-b2g/gaia/blob/master/shared/js/template.js#L122

What was the issue you saw when you tried to remove the argument ?